src/queries/user.ts

```diff
@@ -62,6 +62,9 @@ export const Qu_userGenTokens = gql`
         thumbnailUri
         labels
         lockEnd
+	objkts {
+	  version
+	}
```

Comment:
@ciphrd is there a smarter way to have the version available for the tokens?

Reply:
Damn, I forgot to add a `version` field at the indexer level. We would definitely need to so that this query is optimized before pushing it to prod. It's a small change but requires a re-indexing, so in the meantime I'll keep this PR opened until the changes are made at the indexer's level.